src/index.tsx

```diff
@@ -1,3 +1,5 @@
+'use client'
```

Comment:
Thought: presence of `'use client'` here seems not to change anything for me

Reply:
Double checked that and You're right. Reverted that